### DISCUSSION TITLE
빌드잇에서 만든 맵을 월드로 가져올 수 있나요?

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#64] [5. Q&A]
제페토 빌드잇에서 만든 맵을 제페토 월드로 가져와서 배경으로 사용하고 싶은데,
어떻게 하는 지 좀 알려주세요.

**Comment by x4cf** [comment]
언젠가 그런 기능이 나올거라는 답장은 본 기억은 있지만 아직은 불가능합니다.

페키지 메니저에서 빌드잇 오브젝트를 다운로드 받아서 맵을 만들어보는건 어떨까요?
<img width="853" alt="스크린샷 2021-12-11 오전 10 50 33" src="https://user-images.githubusercontent.com/90679926/145659827-b64039e4-a93c-48ac-bb42-a468af14c200.png">
.

**Reply by runtostar** [reply]
답변 감사드립니다. 그런데 패키지 매너저에 빌드잇 오브젝트가 보이질 않네요.
빌드잇 오브젝트를 따로 가져와야 하는건가요?
![이미지 1](https://user-images.githubusercontent.com/95363110/145661075-93035792-0995-43bc-92c8-77dd1cac0c9c.png)

**Reply by x4cf** [reply]
왼쪽 상단에 보시면 + 버튼  오른쪽이 "페키지:인 프로젝트"로 되어있네요.
나의 레지스트리로 변경하면 나온답니다 :)

**Reply by runtostar** [reply]
역시 고수님이세요~!!!
감사합니다. 정말 정말 감사합니다~!!!

**Comment by hyeonjin-jo** [comment]
안녕하세요

빌드잇에서 만든 맵을 활용 가능한 문서가 업데이트되어서 공유드립니다.

https://studio.zepeto.me/guides/loading-buildit-maps
